js/ui/pivot_grid/ui.pivot_grid.js

```diff
@@ -1167,6 +1167,14 @@ const PivotGrid = Widget.inherit({
         return this.callBase() && !this._dataController.isLoading();
     },
 
+    getUseNativeOptionValue: function() {
```

Comment:
getIsNativeScrollbarInDataArea ?
(seems to be correct inglish)

Reply:
ok, I simplified this logic and passed it to dataArea component